packages/s3-access-metrics/README.md

```diff
@@ -0,0 +1,87 @@
+# S3 Server Access Logs to Cloudwatch Metrics
+
+This package will take
+[S3 Server Access Logs](https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerLogs.html)
+and write them out to Cloudwatch Metrics.
+
+## How it works
+
+When deployed, this will configure a trigger which watches a specified S3 bucket
+for the arrival of new S3 Server Access Logs. When a new log file is uploaded,
+it invokes a Lambda function which parses that log file and writes the number of
+successful and failed GET.OBJECT requests to Cloudwatch Metrics.
+
+The metrics will be available in Cloudwatch under:
+
+* `Namespace`: CumulusDistribution
+* `Metric names`: `SuccessCount` & `FailureCount`
+  * Any S3 request event in the log with a status of 200 will be reported under
+    `SuccessCount`. Any other status code will be reported under `FailureCount`.
+* `Dimensions`: `Stack` = the stack name specified at deployment time
+
+## Deployment Configuration
+
+Before deployment, a `config.yml` must be created.
+
+**Required Parameters**
+
+* `prefix` - a unique prefix to be applied to the CloudFormation stack name, and
+  to any named resources that will be created.
+* `logsBucket` - the S3 bucket where the S3 AccessLogs are being written
+* `stack` - the name of the Cumulus stack associated with this deployment
+
+**Optional Parameters**
+
+* `logsPrefix` - the S3 key prefix of the S3 AccessLogs
+* `deploymentBucket` - the bucket to store deployment artifacts in. If not
+  specified, a bucket will be created and used
+* `permissionsBoundary` - an IAM permissions boundary to be used when managing
+  IAM resources during deployment
+* `vpcId` - the VPC to deploy Lambda functions to. If this is set, `subnetIds`
+  must also be set
+* `subnetIds` - a list of subnets to deploy Lambda functions to. If this is set,
+  `vpcId` must also be set
+
+**Example config.yml**
+
+```yaml
+prefix: my-prefix
+logsBucket: my-logs-bucket
+logsPrefix: path/to/my/logs/
+stack: my-cumulus-stack
+deploymentBucket: my-deployment-bucket
+permissionsBoundary: arn:aws:iam::123456789012:policy/SomeRoleBoundary
+vpcId: vpc-123
+subnetIds:
+  - subnet-123
+  - subnet-456
+```
+
+## Deployment
+
+The `s3-access-metrics` package is deployed using the
+[Serverless Framework](https://serverless.com/framework/docs/providers/aws/guide/).
+
+Deployment has only been tested using Node 8.10.
+
+These instructions assume that you have [Node.js](https://nodejs.org/)
+installed. If you don't, the easiest way to install it is typically to use the [Node Version Manager](https://github.com/nvm-sh/nvm).
+Installation instructions for installing `nvm` can be found [here](https://github.com/nvm-sh/nvm#installation-and-update).
+Once `nvm` has been installed, node v8.10 can be installed by running
+`nvm install 8.10`, followed by `nvm use 8.10`.
+
+**Deployment steps**
+
+If you have not done so already, create a `config.yml` file. Once you have a
```

Comment:
I know that the intent here is to make more independently deployable units, but at least while this package is managed as part of the Cumulus monorepo, I don't love the user experience of having to maintain a separate config file for this package that restates existing deployment config. 

If someone declares this package as a dependency from their deployment repo, they'll already have a top level config file, but then they'll have to add another one for this package.

Reply:
Would the same argument apply to the Cumulus Dashboard, then?

Reply:
I suppose it's debatable, but I don't think so. The dashboard is an independent entity from a Cumulus deployment whereas this component is part of a Cumulus deployment.

Also, the config for the dashboard is mostly just pointing it at the API URL for your Cumulus deployment. Very little, if any, Cumulus deployment config is configured for the dashboard.

Perhaps this is something that shakes out as we refine the process for integrating independent components. But until then, it feels like creating work for users to ask them to create separate config and run a separate deployment command.

Reply:
@bilts Can you weigh in here?

Reply:
To be clear, I like the fact that this package was written as an independently deployable unit and agree that it's a positive direction to move in. I'm just somewhat concerned about the user experience of deploying it given the current state of our tooling. 

However, I think these concerns are probably outside the scope of the actual work being done here, so I don't think they should block merge.

Reply:
If it doesn't block merge, can we capture the work to make the deployment easier for users (re-using cumulus config or whatever).

Reply:
I've created a ticket in JIRA to address this.